### PR TITLE
docs: removed openjs reference

### PR DESCRIPTION
**Pull request by Akshaybagai52**
**GitHub Issue:** #664

**Summary**:

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

Yes/No
[JUSTIFICATION]

### CODE REVIEW

**Comment by Akshaybagai52** [comment]
Hey @benjagm I've removed some of the reference but there are some lines in which I need clarification do I need to remove all the lines covering openjs or openjsf or I can replace openjs with some other keyword
Thanks

File: docs/adr/2021-08-26-code-of-conduct.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ We want to make people feel welcome, and so should have a formal CoC.
 ## Decision Drivers
```
Comment (benjagm) [inline docs/adr/2021-08-26-code-of-conduct.md:15]: We should remove this file from this pr and leave intact. Architectural decision records can't be modified.

File: docs/openjsf/application.md [hunk 2]
```diff
@@ -31,20 +31,6 @@ Our expectations include
...
-"We believe that it is essential to provide a neutral home for critical projects, with common principles of technical governance and accountability." - part of the OpenJSF mission statement.
-
-We, the JSON Schema collaborators, also believe this.
-
-We feel the approach laid out in the CPC charter is reasonable.
-
-We identify with the need for collaboration between JavaScript, web solutions, and related technologies.
-
-
 ## Impact and users of the project
```
Comment (benjagm) [inline docs/openjsf/application.md:34]: We shouldn't modify this file without checking with @Relequestual. Let me check with him.

**Review by benjagm — CHANGES_REQUESTED** [review]
Leave some comments. We need to check with Ben Hutton to know what to do with the application.md file. Probably we'll directly remove the file.

File: docs/adr/2021-08-26-code-of-conduct.md (hunk 1 above)
Comment (Akshaybagai52) [reply docs/adr/2021-08-26-code-of-conduct.md:15]: Okay

**Comment by benjagm** [comment]
Hi @Akshaybagai52 I think there is nothing to modify as of now. Those 2 files should not be modified as they reflect decisions and agreements.

**Comment by Akshaybagai52** [comment]
> Hi @Akshaybagai52 I think there is nothing to modify as of now. Those 2 files should not be modified as they reflect decisions and agreements.
> 
> Is it ok if we cancel the PR and close this PR? Thanks a lot for your efforts!

Yeah sure I will cancel my PR
Thanks

**Pull request closed without merging**